test/e2e/specs/site-editor/zoom-out.spec.js

```diff
@@ -4,7 +4,7 @@
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
 test.describe( 'Zoom Out', () => {
-	test.beforeAll( async ( { requestUtils, admin, page } ) => {
+	test.beforeEach( async ( { requestUtils, admin, page, editor } ) => {
 		await requestUtils.activateTheme( 'emptytheme' );
```

Comment:
@scruffian We can leave the theme activation/deactvation part in the `beforeAll`/`afterAll`.